gitlab/v4/cli.py

```diff
@@ -382,6 +382,7 @@ def get_dict(
 class JSONPrinter:
     @staticmethod
     def display(d: Union[str, Dict[str, Any]], **kwargs: Any) -> None:
+        _ = kwargs  # Disables PyLint 'unused-argument' error
```

Comment:
It seems like this might make pylint happy and avoid the superfluous dummy variables?
```suggestion
    def display(d: Union[str, Dict[str, Any]], **_kwargs: Any) -> None:
```

Reply:
Done